core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -168,6 +168,8 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
 
   var kafkaController: KafkaController = null
 
+  var brokerToControllerChannelManager: BrokerToControllerChannelManager = null
```

Comment:
How about "ControllerUplinkManager" ?

Reply:
Not sure it would be more self-explanatory than the current name.

Reply:
ok.